### PR TITLE
executor: fix session's idle time for "SHOW PROCESSLIST"

### PR DESCRIPTION
**Pull request by mccxj**
for issue #5594

### CODE REVIEW

**Comment by shenli** [comment]
@mccxj Thanks！

**Comment by shenli** [comment]
@mccxj Please add a unit test case for this PR.
The [mockSessionManager](https://github.com/pingcap/tidb/blob/master/executor/show_test.go#L368) may be helpful.

File: executor/show.go [hunk 1]
```diff
@@ -219,7 +214,7 @@ func (e *ShowExec) fetchShowProcessList() error {
 			pi.Host,
 			pi.DB,
 			pi.Command,
-			t,
+			uint64(time.Since(pi.Time) / time.Second),
```
Comment (tiancaiamao) [inline executor/show.go]: I'm afraid this is not a real fix.
pi.Time maybe uninitialized.... say, 1970.1.1....

**Comment by zz-jason** [comment]
@mccxj any update ?

**Comment by shenli** [comment]
@mccxj Any update?

**Comment by shenli** [comment]
@tiancaiamao @jackysp PTAL

File: executor/show.go (hunk 1 above)
Comment (shenli) [reply executor/show.go]: @mccxj Please address the comment here.

**Comment by jackysp** [comment]
Please resolve the conflicts, @mccxj .

**Comment by jackysp** [comment]
Hi @mccxj , 
@imtbkcat wants to continue your work. He starts a new pr for this issue https://github.com/pingcap/tidb/pull/7765 . I'll close this one. Thanks for your contribution!

**Pull request closed without merging**